src/api-client.ts

```diff
@@ -87,8 +89,9 @@ export class RevAiApiClient {
      * See https://docs.rev.ai/api/asynchronous/reference/#operation/SubmitTranscriptionJob
      * Submit media given a URL for transcription. The audio data is downloaded from the URL.
      * @param mediaUrl Web location of media to be downloaded and transcribed
-     * @param options (optional) Options submitted with the job, see RevAiJobOptions object
```

Comment:
We had an established pattern in our docs to declare `optional` in parens if the field was nullable. I see how `options (optional) Options` looks funny out of context, but I think it was correct according to our conventions.